mods/species/bayliens/tajaran/datum/species.dm

```diff
@@ -16,7 +16,11 @@
 	name_plural = "Tajaran"
 	base_external_prosthetics_model = null
 
-	description = "A small mammalian carnivore. If you are reading this, you are probably a Tajaran."
+	description = "The Tajaran are a mammalian species resembling roughly felines, hailing from Meralar in the Rarkajar system. \
+	While reaching to the stars independently from outside influences, the humans engaged them in peaceful trade contact \
+	and have accelerated the fledgling culture into the interstellar age. Their history is full of war and highly fractious \
+	governments, something that permeates even to today's times. They prefer colder, tundra-like climates, much like their \
+	home world and speak a variety of languages, especially Siik and Akhani."
```

Comment:
```suggestion
	description = "The Tajaran are a mammalian species roughly resembling felines, hailing from Meralar in the Rarkajar system. \
	While they were becoming multiplanetary in their home system, the humans established first contact and engaged them in peaceful trade, \
	and have accelerated the fledgling spacefarers into the interstellar age. Their history is full of conflict and highly fractious governments, a tradition alive and well in the modern era. \
	They prefer colder, tundra-like climates, much like their home world, \
	and speak a variety of languages, most notably Siik and Akhani."
```

Reply:
Especially this one! This one was terrible, but your version works way better!